doc/gh-pages.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+cd ..
+SOURCES="doc examples"
+git checkout gh-pages
+rm -rf _build _sources _static
```

Comment:
It would be great if you can modify this file tho. It would be helpful for us to run a script to automatically update `gh-pages` (before we move to circle-ci)